src/atoms/gv-link.js

```diff
@@ -84,6 +83,12 @@ export class GvLink extends LitElement {
           --ph: var(--gv-link-a--ph, 15px);
         }
 
+        *,
+        *:before,
+        *:after {
+          box-sizing: border-box;
+        }
+
```

Comment:
I'm not really understand the problem... but display is Good on the APIM portal 👍 !

Reply:
The issue was coming from https://github.com/gravitee-io/gravitee-ui-components/pull/277/files#diff-2fe6252d8db6b2b1961e6d546e372b93b4a240658e670326fe0763a524f179fdR130 where we set both a "width" using the `flex` and `margin`, in fact the margins were not taken into account for the with computation, so we end up with a width matching full free space + margins on each side